src/res/failcode.ja.json

```diff
@@ -10,6 +10,7 @@
     "anNumberNe.hebi": "矢印の先にある数字が正しくありません。",
     "anShadeNe": "矢印の方向にある黒マスの数が正しくありません。",
     "anTatamiNe.yajitatami": "矢印の方向にあるタタミの数が正しくありません。",
+    "anTurn.turnaround": "線の曲がった回数が数字と違っています。",
```

Comment:
Same as with the rules file: No translation is preferable to Machine translation